src/app/carbon-estimation/carbon-estimation.component.spec.ts

```diff
@@ -65,25 +65,53 @@ describe('CarbonEstimationComponent', () => {
     expect(component.chartOptions.chart.height).toBe(600 - estimatorBaseHeight - 200 - 100);
   });
 
-  it('should recalculate chart height on window resize, for laptop screen', () => {
+  [0.5, 1, 2, 5].forEach(browserZoomFactor => {
```

Comment:
Jasmine doesn't seem to have built in support for parameterised tests. This solution was suggested on Stack Overflow and in a few Medium posts.

Reply:
I've removed the dependency on browserZoomFactor now so this no longer applies.